### PR TITLE
Drop Python 2 support

### PR DESCRIPTION
**Pull request by ernestask**
Signed-off-by: Ernestas Kulik <ekulik@redhat.com>

### CODE REVIEW

**Comment by mkutlak** [comment]
You can also delete:
https://github.com/abrt/satyr/blob/1f44bb82d018be775d41679cf790a5d85606b54c/python/__init__.py#L5
https://github.com/abrt/satyr/blob/1f44bb82d018be775d41679cf790a5d85606b54c/python/doc/satyr.py#L12
https://github.com/abrt/satyr/blob/44f748634560846a440ff46e9eb32e884a118a2e/tests/python/satyr.py#L8

**Comment by ernestask** [comment]
Will do, thanks.

File: satyr.spec.in [hunk 1]
```diff
@@ -1,26 +1,7 @@
-%{!?python2_sitearch: %global python2_sitearch %(%{__python2} -c "from distutils.sysconfig import get_python_lib; print get_python_lib(1)")}
-
-%if 0%{?fedora} || 0%{?rhel} > 7
-# Enable python3 build by default
```
Comment (mkutlak) [inline satyr.spec.in]: I think this should stay. We want to build `python3-satyr` by default.

Comment (ernestask) [reply satyr.spec.in]: https://github.com/abrt/satyr/blob/b1631124e0571efc6baa1383192d7e41dee4122c/configure.ac#L64

Looks to me like it’s enabled by default anyway.

Comment (ernestask) [reply satyr.spec.in]: https://www.gnu.org/savannah-checkouts/gnu/autoconf/manual/autoconf-2.69/autoconf.html#External-Software

`action-if-not-given` sets `with_python3` to `yes`, so the option is just for disabling it.

Comment (mkutlak) [reply satyr.spec.in]: https://rpm.org/user_doc/conditional_builds.html

It defaults to `--without-python3`.

Comment (ernestask) [reply satyr.spec.in]: Right, seems that we’re talking about completely different things here.

Comment (ernestask) [reply satyr.spec.in]: In this case, it wouldn’t default to anything, no? I removed the %bcond_with{,out} lines.

Comment (ernestask) [reply satyr.spec.in]: In that case, I should probably have also removed the conditional when executing `configure`. Do you think it makes sense to keep the conditionals?

Comment (mkutlak) [reply satyr.spec.in]: Keep the `%bcond_without python3` in this PR. We can remove the with/without python3 conditions in the future.

**Comment by ernestask** [comment]
Updated.

**Pull request merged**